clayui.com/src/components/IconSearch/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React, {useEffect, useState} from 'react';
+
+const spritemap = '/images/icons/icons.svg';
+
+const IconSearch = source => {
```

Comment:
Since this is a react component we typically either call this arg `props` or we will immediately destructure it `({source}) =>`